includes/html/api_functions.inc.php

```diff
@@ -1000,6 +1000,14 @@ function list_alerts(\Illuminate\Http\Request $request)
     }
     $sql .= ' ORDER BY A.'.$order;
 
+    $alert_rule = $_GET['alert_rule'];
+    if (isset($alert_rule)) {
+        if (is_numeric($alert_rule)) {
+            $param[] = $alert_rule;
+            $sql .= ' AND `R`.id=?';
```

Comment:
This needs to be moved above the "order by" sql,  
 else it would be like `ORDER BY... AND `R`.id=`  

Did you test this?

Reply:
sorry, I tested it in v1.48 which has no "order by". I'll move it up.

Reply:
Please always submit patches against the current master. And 1.48 is more than 1 year old, it is not 1 or 2 days old ...